src/wrapper-utils.js

```diff
@@ -2,10 +2,11 @@
 // Tests don't define this variable so fallback to behave like chrome
 export const hasMozProxies = typeof mozProxies !== 'undefined' ? mozProxies : false
 const globalObj = typeof window === 'undefined' ? globalThis : window
-const getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor
 const functionToString = Function.prototype.toString
-const objectKeys = Object.keys
 
+/**
+ * @deprecated use the ContentFeature method instead
+ */
 export function defineProperty (object, propertyName, descriptor) {
```

Comment:
How often is this used still?

Reply:
only by `overloadDescriptor()` inside `DDGProxy`

Reply:
Can you raise a task to make this use the same base code? The one inside contentFeature can use this util or something?